Makefile

```diff
@@ -1,23 +1,29 @@
 .PHONY: check
 check: lint cs tests phpstan
 
+vendor: $(wildcard composer.lock) composer.json
+	composer update
+
+build-cs/vendor:
```

Comment:
This is still buggy - if `build-cs/composer.lock` changes, `make build-cs/vendor` will not update the dependencies correctly.

I wonder if these changes are worth it if it's so hard to get right 🤔